docs/api/en/geometries/ExtrudeGeometry.html

```diff
@@ -75,7 +75,7 @@ <h3>[name]([param:Array shapes], [param:Object options])</h3>
 				<li>depth — float. Depth to extrude the shape. Default is 1.</li>
 				<li>bevelEnabled — bool. Apply beveling to the shape. Default is true.</li>
 				<li>bevelThickness — float. How deep into the original shape the bevel goes. Default is 0.2.</li>
-				<li>bevelSize — float. Distance from the shape outline that the bevel extends. Default is bevelThickness - 0.1.</li>
+				<li>bevelSize — float. Distance from the shape outline that the bevel extends. Default is 0.1.</li>
```

Comment:
I'm afraid this change is not right. The default of `bevelSize` is indeed the given bevel thickness minus `0.1`:

https://github.com/mrdoob/three.js/blob/03efa5a594bc44705a0dda94711a76c8e75f21ae/src/geometries/ExtrudeGeometry.js#L79